src/Compilers/VisualBasic/Test/Emit/Emit/CompilationEmitTests.vb

```diff
@@ -2923,5 +2923,89 @@ End Class
             End Using
         End Sub
 
+        Private Class FailingStream
```

Comment:
Oh dang.  Did I forget to remove the FailingStream implementation.  I don't think it's even used anymore.